bin/download_fasta_utils.py

```diff
@@ -41,116 +51,142 @@ def get_fasta_url(accession, analysis_ftp_field="generated_ftp"):
     for row in reader:
         field_name = query[accession_type]["fields"]
         file_url = row[field_name].split(";")[0]  # Split to take the first FTP link if multiple
-        return file_url
-    return None # no information about this accession in ENA
-
-
-# TODO list errors explicitly, raise ValueError instead of returning None
-@retry(tries=5, delay=15, backoff=1.5) 
-def download_from_ENA_FIRE(accession: str, analysis_ftp_field: str, outpath: str) -> str:
+        if file_url != "":
```

Comment:
empty string evaluates as False in an if condition, and vice versa as True, so you can just do this:
```suggestion
        if file_url:
```

Reply:
Yes, it's true. First, I intentionally wrote it to highlight that this check is for file_url = "" and not file_url = None, but now when I look at it, `if file_url:` seems better to me

Reply:
Added in [linting fixes suggested by Chris](https://github.com/EBI-Metagenomics/MAG-to-assembly-pipeline/pull/4/commits/d4e8bf5ee31950a02c4dae06fbc9734df5b10761)